### PR TITLE
[demos] Update images potinting to personal repos

### PR DESCRIPTION
**Pull request by josunect**
A couple of changes:

### Music store demo

* Copied mimik code into a subfolder
* Created Github action to generate the image and upload to Kiali Quay.io repo. 

In order for this to work, it requires the following secrets: 

- QUAY_USER
- QUAY_PASSWORD

### service spawner

* Just replacing image pointing to personal repos

Fixes [#5283](https://github.com/kiali/kiali/issues/5283)

### CODE REVIEW

File: .github/workflows/build-music-store-image.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        id: build-image
+        run: docker build music-store/mimik/ --file music-store/mimik/build/instance/Dockerfile --tag demo_mimik
+
+      - name: Push To quay.io
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: demo_mimik
+          registry: quay.io/kiali
+          username: ${{ secrets.QUAY_USER }}
```
Comment (leandroberetta) [inline .github/workflows/build-music-store-image.yml:32]: I do have the credentials but I don't have the premissions on this repo to add the secrets (the Settings tab is missing).

@jmazzitelli this is for creating the image and pushing it. 

I don't know if we want to add the secrets to the demo repo. This is an image that is not updated much, so maybe we can just push it to the repo manually.

Thoughts?.

**Review by leandroberetta — COMMENTED** [review]
LGTM, but I have a comment about the Quay credentials.

File: .github/workflows/build-music-store-image.yml (hunk 1 above)
Comment (josunect) [reply .github/workflows/build-music-store-image.yml:32]: Thanks, @leandroberetta !

I think it can be a good idea to just upload the image

Comment (josunect) [reply .github/workflows/build-music-store-image.yml]: Removing github actions file

**Review by leandroberetta — APPROVED** [review]
LGTM

**Comment by josunect** [comment]
> LGTM

@leandroberetta thanks for the review! 
If we are not generating the image, I think the mimik folder can we removed

**Comment by josunect** [comment]
Removed mimik folder, no needed as the image was manually updated.

**Review by leandroberetta — APPROVED** [review]
LGTM

**Pull request merged**